lib/active_fedora/file.rb

```diff
@@ -147,6 +161,7 @@ def to_solr(solr_doc = {}, _opts = {})
     end
 
     def content=(string_or_io)
+      raise "This file has external content.  First call external_url=nil if you want to use content=." if external_content?
```

Comment:
@cjcolvar might be nice to have a `external_content_reset` method that does the setting to nil for you.